libs/blocks/notification/notification.js

```diff
@@ -90,8 +90,15 @@ function decorateFlexible(el) {
     el.querySelector('.background'),
     el.querySelector('.foreground'),
     el.querySelector('.close'),
-  ];
+  ].reduce((acc, curr) => {
+    if (curr) return [...acc, curr];
+    return acc;
+  }, []);
```

Comment:
Why do you need the reduce here? Does innards get mutated somehow within `inner` after it's added to the DOM?

Reply:
I'm also a bit confused on this. Would mapping the options so that any element that doesn't exist gets turned to a simple `''`? Or filter it to only include elements that exist?

Reply:
As @overmyheadandbody suggested, filter should work for this.

```suggestion
  ].filter(Boolean);
```

Reply:
It's accounting for situations where `el.querySelector('.background')` and/or `el.querySelector('.close')` return null -- otherwise "null" gets used as content in the `createTag` call below

Reply:
@chrischrischris Sorry -- for some reason my chrome tab of this PR didn't update to include your comment. Will do! 👍